fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -140,7 +140,8 @@ struct TenantManagementWorkload : TestWorkload {
 					wait(tr->commit());
 				} else if (operationType == OperationType::MANAGEMENT_DATABASE) {
 					ASSERT(tenantsToCreate.size() == 1);
-					wait(success(TenantAPI::createTenant(cx.getReference(), *tenantsToCreate.begin())));
+					wait(success(TenantAPI::createTenant(
+					    cx.getReference(), *tenantsToCreate.begin(), deterministicRandom()->coinflip())));
```

Comment:
Instead of using randomness here, I think it would be better to generate a random value at the start of the function and apply it here. That way you can actually verify that the value matches what you set rather than determining the value by reading the tenant entry.

Reply:
Another way to do this would be in the loop above where `tenantsToCreate` set is getting constructed, you can randomly select a tenant for which encryption is supported (remember in a separate `map<tenantName, bool>`; you pass the flag at the create time as well as validate at the end that created-tenant do have the flag appropriate set.

Reply:
I think this might get a little bit easier if you wait for the tenant group PR, as I think it sets up something to allow per-tenant config here.

Reply:
If that's the case, given this change is not that involved @sfc-gh-nwijetunga lets wait till AJ is done with PR and we can revisit this change later. Thoughts?
Thanks.

Reply:
sure we can wait until AJ is done